sswlib/src/main/java/components/MechArmor.java

```diff
@@ -2569,6 +2582,24 @@ public boolean AllowHarJel()
         return Config.AllowHarJel();
     }
 
+    public double GetHarjelMod(int index) {
+        if (HasHarjel(index, Harjel2Locs)) {
+            return 1.1;
+        } else if (HasHarjel(index, Harjel3Locs)) {
+            return 1.2;
+        }
+        return 1.0;
+    }
+
+    private boolean HasHarjel (int index, List<LocationIndex> locs) {

```

Comment:
This probably shouldn't be duplicated here, I'm trying to think if there was a reason I didn't just move this to `*Loadout.java` instead